movies/src/main/java/se/mulander/cosmos/movies/impl/Movies.java

```diff
@@ -286,12 +289,10 @@ public static Response setMovieStatus(String id, String status)
     public static Response getMoviesWithStatus(String status)
     {
         try {
-            Map<String, Object> param = new HashMap<>();
-            param.put("status", status);
-            List result = Database.getObjects("from Movie WHERE extendedMovie.status = :status", param);
+            List<Movie> result = new MovieDaoImpl().getMoviesByStatus(status);
             if (result.isEmpty()) return Response.status(HttpServletResponse.SC_NOT_FOUND)
                     .entity(new ErrorMessage("Could not fetch movies",
-                            "No movies with status " + status + " was found in the database"))
+                            "No movies with the status" + status +  "was found in the database"))
```

Comment:
varför tog du bort whitespace från strängarna?
resultatet blir ju:
"No movies with the statusIN_LIBRARYwas found in the database"

Reply:
glömde lägga tillbaka whitespace :) (tog bort status ett tag då jag fick för mig att den inte fanns tillgänglig. Ändrar tillbaka)

Reply:
Ändrat